### PR TITLE
Minor tweaks

### PR DESCRIPTION
**Pull request by NickeZ**

### CODE REVIEW

File: scripts/set-new-screen.jlink [hunk 1]
```diff
@@ -0,0 +1,4 @@
+mem 0xe002 1
+w1 0xe002 1
```
Comment (benma) [inline scripts/set-new-screen.jlink:2]: can you also add scripts to set it back to the old one? on raw pcb's it's easy to switch screens.

probably not necessary for the chip as you can't switch that :)

**Review by benma — CHANGES_REQUESTED** [review]
pls run `./scripts/docker_exec.sh ./scripts/format` to fix up the CI error complaining about the formatting.

File: scripts/set-new-screen.jlink (hunk 1 above)
Comment (NickeZ) [reply scripts/set-new-screen.jlink:2]: done!

**Review by benma — APPROVED** [review]
utACK

**Pull request merged**